sjsonnet/src/sjsonnet/Evaluator.scala

```diff
@@ -33,20 +33,24 @@ class Evaluator(parseCache: collection.mutable.HashMap[(Path, String), fastparse
   def visitExpr(expr: Expr)
                (implicit scope: ValScope): Val = try {
     expr match {
+      case Id(pos, value) => visitId(pos, value)
```

Comment:
I assume you re-ordered these to try and speed up lookup for the most common cases. How did you find the order? e.g. was it just a guesstimate, did you instrument it to see which ones are most common, or something else?

Reply:
I gathered statistics from our universe benchmark. It may not be representative for all use cases but the current version doesn't seem to be ordered for performance anyway.